.travis.yml

```diff
@@ -41,6 +41,7 @@ before_install:
 
 install:
   - travis_retry composer install --no-interaction --ignore-platform-reqs
+  - php bin/update_hostname_validator.php --check-only
```

Comment:
Block the execution of normal scripts. This is there is no way of to know if any other PR in the queue is well written or not